app/controllers/course/assessment/question/scribing_controller.rb

```diff
@@ -89,7 +89,7 @@ def render_success_json(message)
   end
 
   def render_failure_json(message)
-    render json: { message: message, errors: @scribing_question.errors.full_messages },
+    render json: { message: message, errors: @scribing_question.errors },
```

Comment:
Hmm are we making this switch throughout? I think there are a few other places that utilise `full_messages` as well.